modules/command-engine/core/src/test/scala/surge/internal/persistence/PersistentActorSpec.scala

```diff
@@ -610,5 +610,26 @@ class PersistentActorSpec
       doSerde(PersistentActor.ApplyEvent[String](UUID.randomUUID().toString, "test2"))
       doSerde(eventEnvelope(CountIncremented(UUID.randomUUID().toString, 1, 1)))
     }
+
+    "Serialize/Deserialize ACK types from Akka" in {
+      import akka.serialization.SerializationExtension
+
+      def doSerde[A](envelope: ACK): Unit = {
```

Comment:
This `doSerde` function is duplicated across these few tests for serializing/deserializing various types (request & response) - I think we can extract it to a `def doAkkaSerde(message: AnyRef): Assertion` to eliminate a bit of the duplication